src/Confluent.Kafka/KafkaException.cs

```diff
@@ -23,12 +23,19 @@ namespace Confluent.Kafka
 {
     public class KafkaException : Exception
     {
-        public KafkaException(Error error, string message)
-            : base(message)
+        private Error _error;
```

Comment:
this could be done without explicitly specifying the private member here (as in the existing code). The only thing this can't give you is immutability (i.e. making it readonly, which is another thing you could do), though it's such a trivial class i'd prefer terseness over that. Also coding convention is to not use _'s. I probably agree with you _'s is better, even though that is not official C# coding standard, but if we're going to do that we should do it across the entire codebase.